build.gradle

```diff
@@ -273,6 +273,8 @@ dependencies {
   implementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
   implementation group: 'org.mapstruct', name: 'mapstruct', version: versions.mapstruct
 
+  implementation group: 'org.hibernate', name: 'hibernate-envers', version: '5.4.30.Final'
+
   compile group: 'org.flywaydb', name: 'flyway-core', version: '7.7.0'
```

Comment:
It's not related to your PR. But could you remove unwanted testContainers dependencies from Gradle please. 
https://github.com/hmcts/cpo-case-payment-orders-api/blob/master/build.gradle#L295-L296

Reply:
Why are they unwanted ?

Reply:
we are spinning up container using datasource url (which requires postgres-container dependency only, that pulls automatically core).

Reply:
Ah right, ok - fixed in  6220d88